src/runtime/internal/Component.ts

```diff
@@ -67,7 +31,10 @@ export function mount_component(component, target, anchor, customElement) {
 		add_render_callback(() => {
 
 			const new_on_destroy = on_mount.map(run).filter(is_function);
```

Comment:
Maybe potentially `on_mount` also should change to `component.$$.on_mount` ?

I think execution timing between the param function of `add_render_callback` and others is different.
So I think the param function of `add_render_callback` should not use destructed assignment value.

Reply:
done!